pkg/controllers/validatingadmissionpolicy-generate/controller.go

```diff
@@ -11,11 +11,12 @@ import (
 	kyvernov1listers "github.com/kyverno/kyverno/pkg/client/listers/kyverno/v1"
 	"github.com/kyverno/kyverno/pkg/clients/dclient"
 	"github.com/kyverno/kyverno/pkg/controllers"
+	"github.com/kyverno/kyverno/pkg/event"
 	"github.com/kyverno/kyverno/pkg/logging"
 	controllerutils "github.com/kyverno/kyverno/pkg/utils/controller"
 	datautils "github.com/kyverno/kyverno/pkg/utils/data"
 	kubeutils "github.com/kyverno/kyverno/pkg/utils/kube"
-	"k8s.io/api/admissionregistration/v1alpha1"
+	admissionregistrationv1alpha1 "k8s.io/api/admissionregistration/v1alpha1"
```

Comment:
@MariamFahmy98 - do we need to bump to v1beta1?

Reply:
No. Since we are using client 1.28, it can deal with both v1alpha1 and v1beta1.

Reply:
We can stay with v1alpha1 for now, might need an issue to track v1alpha1 removal in Kubernetes.